releasenotes/notes/0.19/bip-improvement-34cd623294a30d83.yaml

```diff
@@ -1,13 +1,13 @@
 ---
 features:
   - |
-    The ``balanced`` objective function of the ``BIPMapping`` no longer
-    requires a ``BackendProperties``. It will use ``BackendProperties`` if
+    The ``balanced`` objective function of the :obj:`.BIPMapping` no longer
+    requires a :obj:`.BackendProperties`. It will use :obj:`.BackendProperties` if
```

Comment:
Should we update the earlier releae note that says it requires backend props to be set with 2q error rates in the BackendProperties?

Reply:
I've combined this release note into the other, and deleted this one.